change/@uifabric-styling-2020-05-04-17-29-29-fix-cleanup.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Styling tsconfing: adding noImplicitAnies and noUnusedLocals.",
```

Not very important but it was bothering me 😆 
```suggestion
  "comment": "Styling tsconfig: adding noImplicitAny and noUnusedLocals.",
```